be/server/src/main/resources/application.properties

```
@@ -1 +1,6 @@
-
+# logging 관련 설정
+logging.level.com.codesquad.=DEBUG
```

```suggestion
logging.level.com.baseball=DEBUG
```

파이로 실수하셨어요!
이렇게 생각없이 복붙하면 안되요!!!

---

알려줘서 고마워요 로빈!!